man/man8/zpool.8

```diff
@@ -2111,6 +2111,15 @@ Display vdev GUIDs instead of the normal device names. These GUIDs can be used i
 .sp
 .ne 2
 .na
+\fB\fB-k\fR\fR
+.ad
+.RS 12n
+Display the kind of device a pool or vdev is based on: ssd or hdd.  Mixed mirrored devices that have at least one solid-state member are marked "mix".  A pool is considered mixed if all members are at least mixed.  A file vdev is assumed to have ssd characteristics when evaluating parents.
```

Comment:
> Display the kind of device a pool or vdev is based on: ssd or hdd.

I'd suggest adding **file** to this list and any other valid kinds.

> A pool is considered mixed if all members are at least mixed.

This isn't clear to me.  What does _at least mixed_ mean?  I noticed in the example output  the pool itself was identified as **hdd** when I would have expected it to be **mixed**.